### DISCUSSION TITLE
Allow passwordless login with passkeys

### DISCUSSION THREAD

**Discussion by lazytownfan** [simple-login/app#1825] [Feature Requests]
SimpleLogin should allow users to log in on the web without entering a password with [passkeys](https://en.wikipedia.org/wiki/Passkey_(authentication)).

There is an [initiative](https://passkeys.directory/) at `Passkeys.directory` by 1Password to keep a list of which sites support passkeys.

(To be fair, SimpleLogin had implemented WebAuthn security keys before Proton Mail did.)

(Passkeys were mentioned [before](https://github.com/simple-login/app/discussions/1771), but it was only for specifically when using 1Password.)

**Comment by markusqw1** [comment]
This would be great

**Comment by aparatext** [comment]
Are there any updates on this?

Passkeys, especially hardware tokens, aren't meant for 2FA. They are themselves secured by a PIN.

Right now I have to open KeePass, auto-fill password, plug in the Yubikey, and then type its pin and tap the button. If not for how sensitive SimpleLogin is, protecting recovery emails that could reset passwords on key accounts, this UX would have made me switch back to TOTP.
